h/browser/chrome/lib/hypothesis-chrome-extension.js

```diff
@@ -37,6 +37,7 @@
     var store = new h.TabStore(localStorage);
     var state = new h.TabState(store.all(), onTabStateChange);
     var browserAction = new h.BrowserAction(chromeBrowserAction);
+    var pdfHandler = new h.pdfHandler(state);
```

Comment:
Can you make this UpperCamelCase since it's using the `new` keyword and should match the others?